src/framework/filterControl/_filterItem.scss

```diff
@@ -45,3 +47,11 @@
         opacity: 1;
       }
     }
+
+.filterFormDropdown--editing {
```

Comment:
I do understand why it's here, it's more of a structural thing, but no need for changing.